infra/gcp/gcs_bucket/main.tf

```diff
@@ -1,9 +1,9 @@
-resource "random_id" "bucket_prefix" {
-  byte_length = 5
+resource "random_string" "bucket_suffix" {
+  length = 5
 }
 
 resource "google_storage_bucket" "bucket" {
-  name = "${var.short_name}-${random_id.bucket_prefix.hex}"
+  name = "${var.short_name}-${random_string.bucket_suffix.result}"
```

Comment:
This will involve some manual state management by the AUS folks. Probably easy, but we should maybe ask them/give them a heads up. This will try to delete and recreate their buckets if they don't manually create the `random_string` resource to match what their bucket is named.

Reply:
Agreed, worth an email. I found the random_id resource really difficult to import as. If you can give me the incantation that would correctly import our extant buckets, then I'd be happy to keep as is.

Reply:
Have you tried [this](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/id#import)? Seems to suggest you could throw it through base64

Reply:
OK, so, this particular random id is used for buckets of which we only have two: hail_query and hail_batch. In both cases, we created them with the names `hail-query` and `hail-batch`. Those are not importable in either setting. I'll revert this change so that we don't change the gcp folder as much, but we must eventually confront the fact that our naming scheme and the one described in the gcp terraform folder are incompatible.